ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -107,43 +107,11 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         case primary
```

Comment:
Shouldn't `NavigationBar.Style` live in `NavigationBarTokenSet`?

Reply:
The `Style` enum doesn't contain any information inherently about the token values themselves, so I think it's fine where it is now. Let the token set handle these translations.